exercises/security/README.md

```diff
@@ -423,3 +478,133 @@ Practically this means that a single IP can server multiple web services/pages,
 <details>
 <summary>What is a "Backdoor" in information security?</summary><br><b>
 </b></details>
+
+#### Software Supply Chain & Security 
+
+<details>
+<summary>Briefly describe software supply chain. </summary><br><b>
+
+[Increment](https://increment.com/apis/apis-supply-chain-software/): The fundamental idea of the software supply chain is to purchase and implement code solutions for a to-be system. This accelerates the development and deployment of a company’s product. Thus, playing an important role in the software's lifecycle and functionality.
+</b></details>
+
+<details>
+<summary>Give examples of what a code solution could be. </summary><br><b>
+
+  * API's, for instance:
+    * Twilio - provides communication APIs such as information for incoming mobile text messages.
+    * PayPal - provides a payment solution for e.g. e-commerce.
+    * AWS - e.g. outsourcing infrastructures / data centers.
+</b></details>
+
+<details>
+<summary>What're some benefits with software supply chain? </summary><br><b>
+
+[Increment](https://increment.com/apis/apis-supply-chain-software/): Resource-saving. Using and paying for existing solutions to resource-heavy problems saves time as well as money. Hence resulting in efficient, cheap and greater opportunities to develop and deploy software products for consumers. 
+</b></details>
+
+<details>
+<summary>What is a software supply chain attack? </summary><br><b>
+
+[Aquasec](https://www.aquasec.com/cloud-native-academy/devsecops/supply-chain-security/): An attack by an actor who injects malicious software into a part of the software supply chain.
+</b></details>
+
+<details>
+<summary> Give three examples of three potential security threats related to the software supply chain and describe them.</summary><br><b>
+
+[IEEE](https://ieeexplore.ieee.org/abstract/document/9203862): 
+
+  * Sensitive data being exposed or lost.
+    * In a software supply chain, sensitive data may be passed throughout the chain. Security threats involve loss or exposure of this data, such as customer credit card details.
+  * Cloud technology.
+    * Data sharing in the cloud might jeopardize the privacy of the data within the chain.
+  * Third-party vendors.
+    * Third-party vendors’ code solutions might not provide sufficient cybersecurity and risk being a potential subject to data breaches.
+</b></details>
+
+<details>
+<summary> Briefly suggest some solutions related to those threats from a developer perspective. 
```

Comment:
If this is a follow up to another question (let's say the one above) I would explicitly rephrase it as "Following the examples for security threats on supply chain, briefly suggest some solutions related to those threats from a developer perspective"